daemon/libnetwork/libnetwork_linux_test.go

```diff
@@ -1485,11 +1483,11 @@ func TestParallel(t *testing.T) {
 	assert.NilError(t, err)
 
 	sboxes := make([]*libnetwork.Sandbox, numThreads)
-	sboxes[first-1], err = controller.NewSandbox(context.Background(), fmt.Sprintf("%drace", first), libnetwork.OptionUseDefaultSandbox())
+	sboxes[0], err = controller.NewSandbox(context.Background(), fmt.Sprintf("%drace", 1), libnetwork.OptionUseDefaultSandbox())
 	assert.NilError(t, err)
 
-	for thd := first + 1; thd <= last; thd++ {
-		sboxes[thd-1], err = controller.NewSandbox(context.Background(), fmt.Sprintf("%drace", thd))
+	for i := range numThreads {
+		sboxes[i], err = controller.NewSandbox(context.Background(), fmt.Sprintf("%drace", i+1))
```

Comment:
Previously `thd` would start with `first + 1 => 1 +1 => 2`.

Then `sboxes[thd-1] => sboxes[2-1] => sboxes[1]`

Now `for i := range numThreads {` will start with `i = 0`, which would mean `sboxes[0]` instead of `sboxes[1]`.

Reply:
oh! you're right; I should skip the first one here

Reply:
I reverted it.